boards/arm/stm32h747i_disco/stm32h747i_disco_m7.dts

```diff
@@ -54,3 +54,23 @@
 	current-speed = <115200>;
 	status = "okay";
 };
+
+/* From UM2411 Rev 4:
+   With the default setting, the Ethernet feature is not working because of
+   a conflict between ETH_MDC and SAI4_D1 of the MEMs digital microphone */
+&mac {
+	status = "okay";
+	pinctrl-0 = <&eth_ref_clk_pa1
+		     &eth_mdio_pa2
+		     &eth_crs_dv_pa7
+		     &eth_mdc_pc1
+		     &eth_rxd0_pc4
+		     &eth_rxd1_pc5
+		     &eth_tx_en_pg11
+		     &eth_txd0_pg13
+		     &eth_txd1_pg12>;
+};
```

Comment:
Configuration looks good to me ! 👍 

![image](https://user-images.githubusercontent.com/1338015/97783862-bfb29f00-1b9a-11eb-90cc-20f10717f5d2.png)
![image](https://user-images.githubusercontent.com/1338015/97783781-34d1a480-1b9a-11eb-98fa-c7c8b41eaf15.png)
![image](https://user-images.githubusercontent.com/1338015/97783809-5894ea80-1b9a-11eb-9507-dfc95751e598.png)